### PR TITLE
Adds chumbis

### PR DESCRIPTION
**Pull request by DaxDupont**
[why]: Because we need more of these memes
Added chumbis by request https://tgstation13.org/phpBB/viewtopic.php?p=364841#p364841
@BeeSting12 
@bgobandit

### CODE REVIEW

**Comment by nicbn** [comment]
Can we have -10 on PRB for brain damage memes?

**Comment by DaxDupont** [comment]
@nicbn don't be a chumbis

**Comment by PKPenguin321** [comment]
you a chumbis

**Comment by Bawhoppen** [comment]
New brain damage lines aren't funny, especially when they aren't even truly memorable ones

**Comment by PKPenguin321** [comment]
excuse me chumbis but who asked u

**Comment by Bawhoppen** [comment]
brain damage lines should require a joint approval from all maintainers, kor, MSO, and also have a waiting period of 1 year

**Comment by DaxDupont** [comment]
@Bawhoppen please keep going i need more lines that i can copy paste into brain damage lines

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -62,7 +62,11 @@
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
-        "DON'T EVER TUCH ME"
+        "DON'T EVER TUCH ME",
+        "@pick(semicolon)YOU A CHUMBIS",
+        "@pick(semicolon)im 14 y u asking??????",
```
Comment (nicbn) [inline strings/brain_damage_lines.json]: Doesn't even makes sense

File: strings/brain_damage_lines.json [hunk 2]
```diff
@@ -62,7 +62,11 @@
...
+        "can u make me a revanant or red ghoast??!?!",
```
Comment (nicbn) [inline strings/brain_damage_lines.json]: Destroys my immersion

File: strings/brain_damage_lines.json [hunk 3]
```diff
@@ -62,7 +62,11 @@
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
-        "DON'T EVER TUCH ME"
+        "DON'T EVER TUCH ME",
+        "@pick(semicolon)YOU A CHUMBIS",
```
Comment (nicbn) [inline strings/brain_damage_lines.json]: No, you a chumbis

**Comment by Bawhoppen** [comment]
I want to make it clear I dislike stupid brain damage lines even if they're not from goof

**Comment by Tlaltecuhtli** [comment]
these arent even funny

**Comment by Armhulen** [comment]
chumbis is good

the rest are garrrrrrrrbage

**Comment by DaxDupont** [comment]
Depending on what a maintainer wants I can remove everything but chumbis.

**Comment by feemjmeem** [comment]
you a chumbis if you don't merge 'you a chumbis'

**Comment by Qbopper** [comment]
you a chumbis should get in for sure, instant classic, but the others... ehh

**Comment by Bawhoppen** [comment]
People here a joke and then are like "this is really funny, it should be a brain damage line!" Then it gets worn out and is super boring after the first week.
Only the most memorable stuff should be a line.

**Comment by duncathan** [comment]
chumbis is the only worthwhile

**Comment by MMMiracles** [comment]
image

**Comment by optimumtact** [comment]
sigh

**Comment by PKPenguin321** [comment]
Riot

**Comment by DaxDupont** [comment]
@optimumtact is a chumbis

**Comment by Limed00d** [comment]
oranges a chumbis

**Comment by Spaceinaba** [comment]
oranges a chumbis

**Comment by CitrusGender** [comment]
Riot

**Comment by UncleBourbon** [comment]
oranges is a chumbis

**Comment by feemjmeem** [comment]
oranges a chumbis

**Comment by Pascal125** [comment]
Orange the Chumbis

**Comment by PKPenguin321** [comment]
high chumbis density detected in citrus region

**Pull request closed without merging**